hz: 15
trans_upper_left: 0.333
trans_upper_right: 0.666
line_grad_th: 0.666
line_length_th: 10
close_line_th: 7
cand_hsv_lo: [10,0,230] #[0,0,210]
cand_hsv_hi: [30,5,255] #[180,70,255]
rect_w_lo: 170 #150
rect_h_lo: 25  #10
rect_h_hi: 40  #30
whiteness_th_lo: 7.0
whiteness_th_hi: 100.0
visualize: True
